trusted-committer-article.md

```diff
@@ -0,0 +1,232 @@
+      board with the purpose.
+    - Laude excellent or first contributions
+    - Make sure community members have a chance to level up by suggesting 
+      challenging tasks.
+    - Commmunicate technical and community risks to responsible managers
+    - Help potential community members with them beeing empowered to make
+      contributions by talking to their managers
+    - Ensure there are users and potential contributors by promoting and
+      advertising the community and its goals
+    - Make sure people get a together regularly
+      
+## Reducing Barriers To Entry
+
+- Soliciting contributions is one of the things that is more challenging with
+  InnerSource compared to Open Source. There are two main reasons for that 
+    - The sheer number of potential contributors is lower in InnerSource
+    - Contributors will want to contribute during their work time. That means
+      they are more time constrained compared to doing Open Source after office
+      hours.
+- It is therefore super important to make the process for making contributions
+  and for onboarding contributors as frictionless as possible to avoid wasting
+  a contributors time. 
+- There are a number of things a TC can do to achieve that state
+    - have a good readme.md, providing detailed instructions on how to get,
+      build, test and use the SW developed by the InnerSource community.
+    - have a good contribute.md which outlines what is expected of the
+      contributor to make a contribution. It should answer common questions
+      like
+        - Who do I contact in case of questions and how to reach her?
+        - How do I submit a bug report or feature request?
+        - What is the definition of done for a contribution; code, tests, docs?
+        - What are conventions for code style, branching or committ messages, e. g.?
+        - What is expected of me in terms of supporting contributed code after
+          the contribution was accepted?
+        - What are the process steps that govern contributions?
+        - Is there a code of conduct or other guidelines to how the community
+          operates?
+- As a TC, I have to make sure there is someone in the community who is
+  available to answer questions of users and potential contributors.
+- Actively publishing a list of "Newbie Tasks" to actively solicit contributions,
+  rather than waiting for them to happen.
+- Also make it clear that non-code contributions are welcome, such as writing
+  documentation, organizing events or creating artwork.
+- Gig-Marketplace as a good example of how to do this.
+
+## Uplevelling Community Members
+
+- identifying talent
+- encouraging growth
+- mentoring community members (e. g. during PRs)
+
+## Advocating The Communities Needs
+
+- One major difference of InnerSource compared to Open Source software
+  development is the degree of freedom communities have. 
+- Naturally, an InnerSource community is more constrained because it has to fit
```

Comment:
This is a great point, but not sure what it is you are trying to get at.  I think this one leads to the next (that there are varying degrees of freedom depending on the end goal. 

Perhaps it is useful for the TC to identify this end goal early on, so that they can more effectively guide contributions.

Reply:
What I am trying to do is to motivate why the context that InnerSource exists in (corporate) by its very nature makes it likely that conflicts btw. a communities need and the corporations interests might have to be resolved.

Can you elaborate what you are referring to with _end goal_, @mrsanz ?

Reply:
I think what I was trying to say was that these three items:

- One major difference of InnerSource compared to Open Source software development is the degree of freedom communities have.
- Naturally, an InnerSource community is more constrained because it has to fit in the corporate context.
- There are varying degrees of freedom, depending on whether your InnerSource initiative focuses more on permissionless innovation or on shared development of reusable components.

seem like one. I think you could revise them to be more concise.  The end goal of all of these statements is what?  I'd take it one step further.  e.g.

> InnerSource communities exist within the corporate context, will have varying degrees of freedom within your organization.  This means as a Trusted Committer it is important to be strategic (e.g. working within the degrees of freedom of your organization) when advocating for the communities needs.

Reply:
> The end goal of all of these statements is what?

What I was trying to convey is how InnerSource differs from OpenSource and is _more_, rather than _less_ complex as a result.

Reply:
I like your point about how TCs have to be strategic, @mrsanz !

Reply:
I will condense my three bullet points and add your point about a TC needing to be strategic at the end of this section, @mrsanz .